File: dart/com/spoonacular/client/model/inline_response20047.dart
part of openapi.api;

class InlineResponse20047 {
  
  List<InlineResponse20047RecommendedWines> recommendedWines = [];
  
  int totalFound = null;
  InlineResponse20047();

  @override
  String toString() {
    return 'InlineResponse20047[recommendedWines=$recommendedWines, totalFound=$totalFound, ]';
  }

  InlineResponse20047.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['recommendedWines'] == null) {
      recommendedWines = null;
    } else {
      recommendedWines = InlineResponse20047RecommendedWines.listFromJson(json['recommendedWines']);
    }
    if (json['totalFound'] == null) {
      totalFound = null;
    } else {
          totalFound = json['totalFound'];
    }
  }

  Map<String, dynamic> toJson() {
    Map <String, dynamic> json = {};
    if (recommendedWines != null)
      json['recommendedWines'] = recommendedWines;
    if (totalFound != null)
      json['totalFound'] = totalFound;
    return json;
  }

  static List<InlineResponse20047> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20047>() : json.map((value) => new InlineResponse20047.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20047> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineResponse20047>();
    if (json != null && json.isNotEmpty) {
      json.forEach((String key, dynamic value) => map[key] = new InlineResponse20047.fromJson(value));
    }
    return map;
  }
}

File: csharp/src/main/CsharpDotNet2/spoonacular/Model/InlineResponse20047.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace spoonacular.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20047 {
    /// <summary>
    /// Gets or Sets RecommendedWines
    /// </summary>
    [DataMember(Name="recommendedWines", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "recommendedWines")]
    public List<InlineResponse20047RecommendedWines> RecommendedWines { get; set; }

    /// <summary>
    /// Gets or Sets TotalFound
    /// </summary>
    [DataMember(Name="totalFound", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "totalFound")]
    public int? TotalFound { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20047 {\n");
      sb.Append("  RecommendedWines: ").Append(RecommendedWines).Append("\n");
      sb.Append("  TotalFound: ").Append(TotalFound).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
